app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt

```diff
@@ -0,0 +1,88 @@
+package io.homeassistant.companion.android.settings.wear.views
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mikepenz.iconics.typeface.library.community.material.CommunityMaterial
+import io.homeassistant.companion.android.common.data.prefs.impl.entities.TemplateTileConfig
+import io.homeassistant.companion.android.settings.views.SettingsRow
+import io.homeassistant.companion.android.common.R as commonR
+
+@Composable
+fun SettingsWearTemplateTileList(
+    templateTiles: Map<Int, TemplateTileConfig>,
+    onTemplateTileClicked: (tileId: Int) -> Unit,
+    onBackClicked: () -> Unit
+) {
+    Scaffold(
+        topBar = {
+            SettingsWearTopAppBar(
+                title = { Text(stringResource(commonR.string.template_tiles)) },
+                onBackClicked = onBackClicked,
+                docsLink = WEAR_DOCS_LINK
+            )
+        }
+    ) { padding ->
+        Column(
```

Comment:
This column should be scrollable (vertical) in case of a short screen/large text and multiple tiles to ensure they all can be edited. Make sure to apply scroll after setting padding, otherwise you end up with a white/black border around your scrollable list :)

Reply:
Makes sense :+1: Would this be the right way of doing that?

```diff
diff --git app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt
index 24b927e1..b4d33273 100644
--- app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt
+++ app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt
@@ -2,6 +2,8 @@ package io.homeassistant.companion.android.settings.wear.views
 
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
 import androidx.compose.material.Scaffold
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
@@ -33,6 +35,7 @@ fun SettingsWearTemplateTileList(
             Modifier
                 .padding(padding)
                 .padding(all = 16.dp)
+                .verticalScroll(rememberScrollState())
         ) {
             if (templateTiles.entries.isEmpty()) {
                 Text(stringResource(commonR.string.template_tile_no_tiles_yet))
```

Reply:
Sorry, wrong way around :) Apply the scroll first, then the padding. If you scroll now, it'll have the borders I mentioned because you apply the scrolling modifier to what remains after applying the padding.

Reply:
Alright :+1: 

```diff
diff --git app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt
index f6fb0fb3..423ea159 100644
--- app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt
+++ app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt
@@ -38,9 +38,9 @@ fun SettingsWearTemplateTileList(
     ) { padding ->
         Column(
             Modifier
+                .verticalScroll(rememberScrollState())
                 .padding(padding)
                 .padding(all = 16.dp)
-                .verticalScroll(rememberScrollState())
         ) {
             if (templateTiles.entries.isEmpty()) {
                 Text(stringResource(commonR.string.template_tile_no_tiles_yet))
```